store/types/store.go

```diff
@@ -38,6 +38,48 @@ type Queryable interface {
 //----------------------------------------
 // MultiStore
 
+// StoreUpgrades defines a series of transformations to apply the multistore db upon load
+type StoreUpgrades struct {
+	Renamed []StoreRename `json:"renamed"`
+	Deleted []string      `json:"deleted"`
+}
+
+// StoreRename defines a name change of a sub-store.
+// All data previously under a PrefixStore with OldKey will be copied
+// to a PrefixStore with NewKey, then deleted from OldKey store.
+type StoreRename struct {
+	OldKey string `json:"old"`
```

Comment:
```suggestion
	OldKey string `json:"old_key"`
```

Reply:
Actually I don't like the long json names. I would make the golang names shorter as well, but I don't like using `New` as a variable name.

Reply:
The name isn't too verbose. I generally find it to be nice when YAML/JSON tags line up with  their respective member names. imho `old_key` and `new_key` read better.

I can approve once these changes are made 👍

Reply:
Done